### PR TITLE
Add install guide for Arch Linux

### PR DESCRIPTION
**Pull request by Aloxaf**
I have submitted it to AUR so that Arch Linux users can install it easyily.  :D

### CODE REVIEW

**Comment by kimci86** [comment]
Thank you for making this package!
However, before merging your change, I would like to make it clear I do not manage this package myself. Could you rephrase your change?

**Comment by Aloxaf** [comment]
> Thank you for making this package!
> However, before merging your change, I would like to make it clear I do not manage this package myself. Could you rephrase your change?

You're welcome.
I have marked it as unoffcial, is it OK?
